src/ComponentFolders/SignUpPage/SignUp.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import Image from './Images/SofaImage.svg';
+import lineImage from './Images/line.svg';
+import FacebookLogo from './Images/FacebookLogo.svg';
+import GoogleLogo from './Images/GoogleLogo.svg';
+
+function SignUp() {
+  const [signUpData, setSignUpData] = useState({
+    userFirstName: '',
+    userLastName: '',
+    userEmail: '',
+    userConfirmEmail: '',
+    userPassword: '',
+    userCondirmPassword: '',
+    dayOfBirth: '',
+    monthOfBirth: '',
+    yearOfYear: '',
+  });
+
+  function handleOnClick(e) {
+    const { value, name } = e.target;
+    return setSignUpData((prevObj) => {
+      return {
+        ...prevObj,
+        [name]: value,
+      };
+    });
+  }
+
+  return (
+    <div className="h-screen flex justify-center content-center md:flex-wrap max-[767px]:flex-wrap gap-x-20">
```

Comment:
![Screenshot_20221228_193321](https://user-images.githubusercontent.com/110013797/209843537-51ccef67-836a-4a89-a08f-9b1b9fa4c433.png)
if it is possible for you to make those input fields go under eachother with smaller screen it would be great :D

Reply:
I tired but it would not be better, I think the current style is okay